### PR TITLE
Add error handling how-to to README

### PR DESCRIPTION
**Pull request by marcondag**

### CODE REVIEW

File: controllers/DisputesController.js [hunk 1]
```diff
@@ -173,7 +173,7 @@ const DisputesController = Class('DisputesController').inherits(RestfulControlle
       } catch (e) {
         Raven.captureException(e);
         logger.error(
-          `Error occured while updated dispute data for dispute ${dispute.id}`,
```
Comment (shoshber) [inline controllers/DisputesController.js]: Can you change `updated` to `updating`, since you're editing this line anyway?

**Comment by shoshber** [comment]
Thanks!!

File: controllers/DisputesController.js (hunk 1 above)
Comment (marcondag) [reply controllers/DisputesController.js]: Oh, definitely idk why I didn't catch that too!

Comment (shoshber) [reply controllers/DisputesController.js]: we're not hired for our copy-writing skills :laughing:

**Pull request merged**